src/cryptoadvance/specter/templates/welcome/components/tick_checkboxes.jinja

```diff
@@ -86,7 +86,7 @@
                 <td style="text-align: left;">
                 </td>
             </tr>
-            {% if specter.chain == 'main' %}
+            {% if specter.chain == 'main' and specter.node.node_type == "BTC "%}
```

Comment:
A `SpectrumNode` does have a `node_type` which is not `BTC`? I'm confused here.

Reply:
![grafik](https://user-images.githubusercontent.com/70536101/212751684-0dc07e2d-23e6-4ebf-aa1a-8a4246387505.png)